### ISSUE TITLE
Checkbox - Missing Grouping Element (Fieldset)

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1221]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/a0e85a10-5d37-11ee-b395-1337512b2ee2?activeTab=dt-issue&page=0&sortField=ordinal&sortDir=asc&row=3&issuesCount=&issuesCount=&filter=

## Description

The group label is not associated with its checkboxes. The purpose of individual checkboxes in the group cannot be understood without the group label.

Element description 1: The  group label for the following field is "Checkbox List Label"

- Unchecked Checkbox
- Checked Checkbox
- Unchecked Disabled Checkbox
- Checked Disabled Checkbox
- Error Checkbox
